### PR TITLE
Update BarVaryingLoad

### PR DESCRIPTION
**Pull request by IsakNaslundBh**
### NOTE: Depends on 

https://github.com/BHoM/BHoM/pull/1075
https://github.com/BHoM/BHoM_Engine/pull/2160
   
### Issues addressed by this PR

Aligns with changes to the BarVaryingDistributedLoad.

### Test files

https://burohappold.sharepoint.com/:f:/s/BHoM/Eha-mq-0l55KryMeBe543YkBf2M6CQUaOaL6xgKfeFLg6Q?e=rIb1yO

### CODE REVIEW

**Review by alelom — APPROVED** [review]
LGTM

File: SAP2000_Adapter/CRUD/Read/Loads.cs [hunk 1]
```diff
@@ -431,10 +431,10 @@ private List<ILoad> ReadBarVaryingDistributedLoad(List<string> ids = null)
...
                                 loads.Add(new BarVaryingDistributedLoad()
                                 {
-                                    DistanceFromA = dist1[i],
-                                    ForceA = forceA,
-                                    DistanceFromB = dist2[i],
-                                    ForceB = forceB,
+                                    StartPosition = dist1[i],
+                                    ForceAtStart = forceA,
+                                    EndPosition = dist2[i],
+                                    ForceAtEnd = forceB,
```
Comment (jean-lucjacksonBH) [inline SAP2000_Adapter/CRUD/Read/Loads.cs:437]: EndPosition always reports absolute value (meters) when pulled from SAP, but does not set RelativePositions = false (seems to be true by default). Therefore if distances are pushed as relative values they won't match the pulled values, and the Relative flag won't give the user a heads up.

**Review by jean-lucjacksonBH — CHANGES_REQUESTED** [review]

File: SAP2000_Adapter/CRUD/Read/Loads.cs (hunk 1 above)
Comment (IsakNaslundBh) [reply SAP2000_Adapter/CRUD/Read/Loads.cs:437]: Great catch @jean-lucjacksonBH ! Have updated!

**Review by jean-lucjacksonBH — APPROVED** [review]
LGTM!

**Pull request merged**